Forte.EpiServer.AzureSearch/Plugin/ContentIndexingScheduledJob.cs

```diff
@@ -52,17 +54,26 @@ public override string Execute()
             OnStatusChanged("Indexing content start...");
             _contentIndexer.Index(ContentReference.RootPage, indexContentRequest).GetAwaiter().GetResult();
 
+            var exceptionsThresholdReached = indexContentRequest.Statistics.Exceptions.Count >= indexContentRequest.ExceptionThreshold;
+
+            if (exceptionsThresholdReached)
+            {
+                message += $"Exceptions threshold reached. " +
+                           $"Exceptions: {string.Join(NewLine + NewLine, indexContentRequest.Statistics.Exceptions.Select(e => e.ToString()))}{NewLine}{NewLine}" +
+                           $"Failed for ids: {string.Join(",", indexContentRequest.Statistics.FailedIds.Select(c => c.ID))}";
+
+                throw new AggregateException(message);
```

Comment:
`AggregateException` accepts `IEnumerable<Exception>` as a second argument. I feel more readable case would be if message would be build with basic data and all details should go in AggregateException's innerExceptions

Reply:
Passing exceptions to its constructor would hide majority of exceptions' messages and their stack traces in EPiServer's job output. So to both avoid misuse of `AggregateException` and preserve debug information in admin panel I've added custom exception.